jdaviz/configs/imviz/plugins/line_profile_xy/line_profile_xy.py

```diff
@@ -11,7 +11,7 @@
 __all__ = ['LineProfileXY']
 
 
-@tray_registry('imviz-line-profile-xy', label="Imviz Line Profiles (XY)")
+@tray_registry('imviz-line-profile-xy', label="Line Profiles (XY)")

```

Comment:
Typo?
```suggestion
@tray_registry('imviz-line-profile-xy', label="Image Profiles (XY)")
```

Reply:
ah, I had that locally but somehow missed staging it 🤦 .  I'll take another look at the final diff to make sure I didn't miss anything else, they're so easy to miss.

Reply:
Let me know when you done. Thanks!

Reply:
everything looks right to me now, thanks!  Let me know if you see anything else